src/Assets/Fonts.tsx

```diff
@@ -13,6 +13,10 @@ const GaramondSizes = {
     size: "15px",
     height: "1.25em",
   },
+  s16: {
+    size: "16px",
+    height: "1.4em",
+  },
```

Comment:
I added this just because `16px` is used in the Zeplin mockup, but let me know if this is something we should avoid - I feel we might end up adding every single `px` here, which doesn't seem right.

Reply:
Right, keep in mind we want to phase those font sizes out very soon so we'll have to map it to one of the sizes in the new type system